components/Tabs/examples/TabBarTextOnlyExample.m

```diff
@@ -72,6 +87,7 @@ - (void)loadTabBar {
   // rely on the UIAppearance proxy.
   self.tabBar.selectedItemTintColor = [UIColor whiteColor];
   self.tabBar.tintColor = [UIColor colorWithWhite:1.0f alpha:0.5f];
```

Comment:
Tangentially related: any plans to move away from tintColor in MDCTabBar and instead use color properties that have clearer semantic meaning (e.g. barBackgroundColor, disabledTabColor, etc.)? @ianegordon has mentioned tintColor applied to some UIKit components can have unpredictable colorization results.

Reply:
No plans at the moment and they would be independent of this work, but I agree it would help add some uniformity to the API. Should be as easy as adding a selectionIndicatorTintColor property.